### ISSUE TITLE
🐛 BUG: printStats shows Infinity when --experimental-ssr enabled

### ISSUE THREAD

**Issue by leader22** [withastro/astro#2818]
### What version of `astro` are you using?

0.24.3

### What package manager are you using?

npm

### What operating system are you using?

Mac

### Describe the Bug

Stats are shown correctly w/o `--experimental-ssr`.

```
> astro build

10:24:25 AM [build] 2 pages built in 3.18s (1592ms/page)
10:24:25 AM [build] 🚀 Done
```

But w/ `--experimental-ssr`, 

```
> astro build "--experimental-ssr"

10:24:11 AM [build] 0 pages built in 3.06s (Infinityms/page)
10:24:11 AM [build] 🚀 Done
```

It seems `pageCount` is always `0` and `buildTime / 0` = `Infinity` is printed.

https://github.com/withastro/astro/blob/4914e1f70dd08cecb96ca0599b0006e075a50ec1/packages/astro/src/core/build/index.ts#L171

Since this is an experimental flag, please close this if you already aware of. Sorry in that case. 😅 

### Link to Minimal Reproducible Example

N/A

**Comment by FredKSchott** [comment]
cc @matthewp

**Comment by matthewp** [comment]
Yeah that part probably shouldn't be printed at all.

**Comment by natemoo-re** [comment]
AFAICT this was fixed in https://github.com/withastro/astro/pull/2857 and released in `astro@0.25.0`!
